b2sdk/sync/scan_policies.py

```diff
@@ -18,10 +18,7 @@
 
 logger = logging.getLogger(__name__)
 
-try:  # python 3.5 and 3.6 compatibility
-    regex_class = re.Pattern
-except AttributeError:
-    regex_class = re._pattern_type
+regex_class = re.Pattern
```

Comment:
Why not removing `regex_class` and using it directly `re.Pattern` in appropriate place? The variable is redundant now